eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Bbody.hs

```diff
@@ -100,28 +93,12 @@ maxRefScriptSizePerBlock :: Int
 maxRefScriptSizePerBlock = 1024 * 1024 -- 1MiB
 
 data ConwayBbodyPredFailure era
-  = WrongBlockBodySizeBBODY
-      -- | Actual Body Size
-      !Int
-      -- | Claimed Body Size in Header
-      !Int
-  | InvalidBodyHashBBODY
-      -- | Actual Hash
-      !(Hash (EraCrypto era) EraIndependentBlockBody)
-      -- | Claimed Hash
-      !(Hash (EraCrypto era) EraIndependentBlockBody)
+  = WrongBlockBodySizeBBODY !(Mismatch 'RelEQ Int)
+  | InvalidBodyHashBBODY !(Mismatch 'RelEQ (Hash (EraCrypto era) EraIndependentBlockBody))
```

Comment:
These need an explanation comment that clarifies that although the wording may be confusing the `actual` ones correspond to `supplied` and the `claimed` correspond to `exepected` in the `Mismatch` type.